packages/web-components/src/search-widget/client/api/constants.ts

```diff
@@ -0,0 +1,15 @@
+const CONSTANTS: { [key: string]: { [key: string]: string } } = {
+  LOCAL: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
+  },
+  DEV: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
+  },
+  PROD: {
+    WEB_COMPONENT_API_BASE_URL_SEARCH_WIDGET: 'http://localhost:3000',
```

Comment:
So will we change it to ENV later right ?

Reply:
Updated